ClosedXML/Excel/XLTheme.cs

```diff
@@ -18,16 +18,49 @@ internal class XLTheme : IXLTheme
         public XLColor Hyperlink { get; set; }
         public XLColor FollowedHyperlink { get; set; }
 
-        private TypeAccessor accessor = TypeAccessor.Create(typeof(XLTheme));
-
         public XLColor ResolveThemeColor(XLThemeColor themeColor)
         {
-            var tc = themeColor.ToString();
```

Comment:
I'd like to future-proof this method, in case new theme colours are added. I'd prefer to keep this approach, but just rely in reflection.

Reply:
That's why I added a test `CanResolveAllThemeColors` - it will start failing if we forget to add a branch in the switch. The idea is, `ResolveThemeColor` is called way more often that new enum values are added, so it is beneficial to replace reflection with switch-based code.

Reply:
But that will require a new build of ClosedXML. But OK, let's cross that bridge when we get there, if ever.